### ISSUE TITLE
Agent information cannot be parsed

### ISSUE THREAD

**Issue by Damian-Mangold** [wazuh/wazuh#14710]
| Wazuh |
| ----- |
| 4.3.7   |

**Description**
Running the [Release 4.3.7 - Release Candidate 1 - E2E UX tests - Demo environment](https://github.com/wazuh/wazuh/issues/14674), we find [an error message](https://github.com/wazuh/wazuh/issues/14674#issuecomment-1220000738) in the `Master-env2` environment. The message observed are the following:

```
wazuh-authd: ERROR: Invalid request for new agent from: 106.75.218.140
```
Apparently the error occurs when the [agent information cannot be parsed](https://github.com/wazuh/wazuh/blob/master/src/os_auth/auth.c#L114).

An additional piece of information, we cannot see an agent connected with that IP address:

```
[root@wazuh-manager-master-0 ec2-user]# /var/ossec/bin/agent_control -l

Wazuh agent_control. List of available agents:
   ID: 000, Name: wazuh-manager-master-0 (server), IP: 127.0.0.1, Active/Local
   ID: 001, Name: Amazon, IP: any, Active
   ID: 002, Name: Ubuntu, IP: any, Active
```

**Steps to reproduce**
To find the error message, you need to connect to the `Master-env2` environment and run the command: `egrep -i "ERROR|WARNING" /var/ossec/logs/ossec.log`

**Expected Result**
No error

**Actual Result**
Error

**Comment by vikman90** [comment]
Hi @mangolddamian,

Wondering if this happened because the agent had a password set, while the manager had no password. Anyway, enabling debug-2, we can check the log:

```
wazuh-authd: DEBUG: Request received: <...>
```

Which will help us determine the cause of this error.

Cheers.

**Comment by mangolddamian** [comment]
Hello @vikman90 

I did the checks you suggested, here the results.
Before I started I checked the new log and I can't see the reported error:
![image](https://user-images.githubusercontent.com/106940255/186183531-0a51e803-8462-4294-a112-70b82a36ccde.png)

After that I enabled debug-2:
![image](https://user-images.githubusercontent.com/106940255/186182070-83b209ac-657f-4739-9abf-c44e9e5215a9.png)

restart the manager
![image](https://user-images.githubusercontent.com/106940255/186184584-d7e4e923-6146-486c-a3b0-da3697c6eb4f.png)

check status
![image](https://user-images.githubusercontent.com/106940255/186184785-0a2d81d8-a74a-4561-b13f-aa54a1becc85.png)

and again, I can find the reported error.
![image](https://user-images.githubusercontent.com/106940255/186185059-bb138dcc-86b3-40df-846a-6fcf86c9e60a.png)


So, everything seems to indicate that it was a specific error during configuration? If so, could we dismiss the error report? Do you recommend I do more tests?

**Comment by verdx** [comment]
The same error has appeared in a Footprint test for version `4.6.0`, https://github.com/wazuh/wazuh/issues/18922.

**Comment by vikman90** [comment]
Will be fixed in 5.0.
